homeassistant/components/knx/websocket.py

```diff
@@ -213,3 +233,201 @@ def forward_telegram(telegram: TelegramDict) -> None:
         name="KNX GroupMonitor subscription",
     )
     connection.send_result(msg["id"])
+
+
+@websocket_api.require_admin
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "knx/validate_entity",
+        **CREATE_ENTITY_BASE_SCHEMA,  # type: ignore[dict-item]
+    }
+)
```

Comment:
here mypy raised
> error: Unpacked dict entry 1 has incompatible type "dict[Required, ABCMeta]"; expected "SupportsKeysAndGetItem[Marker, Any]"  [dict-item]

I don't think this error is valid, as `Required` inherits from `Marker`, `dict` should support keys and getitem and `Any`... well is any. Am I missing something here?